gui/default/untrusted/syncthing/core/syncthingController.js

```diff
@@ -1993,13 +2015,20 @@ angular.module('syncthing.core')
             });
         };
 
-        $scope.addFolderAndShare = function (folderID, folderLabel, device) {
+        $scope.addFolderAndShare = function (folderID, pendingFolder, device) {
```

Comment:
I suggest porting the argument change to the default GUI as well, so the diff is again only encryption related.